source/configuration_manual/event_filter.rst

```diff
@@ -72,6 +72,7 @@ The value comparison is case-insensitive, but the key is case-sensitive.
 There are some limitations on which operators work with what field types:
 
 * string: Only the ``=`` operator is supported.
+* ip: Only the ``=`` operator is supported.
```

Comment:
could explain semantics bit better, like that you can do `ip = 1.0.0.0/16` and such